model_cards/pradhyra/AWSBlogBert/README.md

```diff
@@ -0,0 +1,9 @@
+This model is pre-trained on blog articles from AWS Blogs. 
+Training code can be found at this [url]()
```

Comment:
Do you forget to add the URL?

Reply:
Sorry - I actually meant to delete that line. It is in a private repo that I'm not able to link to.